server/server.go

```diff
@@ -85,6 +90,31 @@ func (s *Server) LoadGist(ctx *gin.Context) {
 	ctx.String(resp.StatusCode, string(gistContent))
 }
 
+type versionRequest struct {
+	request
+}
+
+func (s *Server) Version(ctx *gin.Context) {
+	req := versionRequest{}
+	req.ServerURL = ctx.Query("server_url")
+
+	cli, err := s.clientForRequest(req.request)
+	if err != nil {
+		ctx.JSON(http.StatusInternalServerError, jsonError("error starting client: %s", err))
+		return
+	}
+
+	resp, err := cli.NewVersionRequest().Do()
+	if err != nil {
+		ctx.JSON(http.StatusInternalServerError, jsonError("error getting server version: %s", err))
+	}
+
+	ctx.JSON(200, map[string]string{
```

Comment:
since the `ParseResponse` and the `VersionResponse` seems to be both `protocol.Response` it could be reused the `toHTTPStatus(resp.Status)` method, and avoid the magic number `200` as already done by `Server.HandleParse(*gin.Context)`

Reply:
This endpoint doesn't just proxy to bblfsh server, but returns dashboard version too. So I don't think it's a good idea to proxy status code. How about just change 200 to `http.StatusOK`?

Reply:
But that's what `toHTTPStatus` seems to do: return an `http.StatusOK` when the `protocol.VersionResponse` succeded, doesn't it?

Reply:
this endpoint is __not__ a proxy. but okay, it doesn't really matter. I'll change.

Reply:
I'm not saying that because it would be a proxy. My point is that both handlers works the same way:
> The browser -> requires something to the server -> the server asks to bblfshd -> bblfshd return a response with a `.Status` -> the server process the bblfsh response and maps the `resp.Status` to a `httpStatus` -> returns a response to the browser.

That's because I see that `toHTTPStatus` suits both handlers.